decidim-core/lib/decidim/commands/create_resource.rb

```diff
@@ -35,22 +33,35 @@ def call
       #  create_resource(soft: true) - Will soft-create the resource, returning any validation errors.
```

Comment:
I could not find any usage for this parameter, and we're rising an error here, so I think we should update the comment and also let possible implementers know that this will be removed in the next version.

```suggestion
      #  create_resource(soft: true) - Used to soft-create the resource. Now it is deprecated and will be removed in decidim version 0.30. 
```

Reply:
Removed in [57efeb5](https://github.com/decidim/decidim/pull/12194/commits/57efeb57cf0a2b0ca49cfd44f3294c1976614377)

Reply:
Great. I thought we arrived to add it to v0.28. As that's not the case, I agree that's better to just remove it.